roles/ceph-osd/tasks/standard.yml

```diff
@@ -4,15 +4,15 @@
   shell: ceph osd crush add-bucket {{ ssd_pool }} root
   register: result
   changed_when: result.stdout |search("added bucket")
-  when: inventory_hostname == groups['ceph_osds_ssd'][0]
+  when: inventory_hostname == (play_hosts | intersect(groups['ceph_osds_ssd']))[0]
```

Comment:
I understand the problem, my question is around how we can preserve nature of run_once even when we have both ssd and hybrid? Want to avoid running it multiple times within a play as other roles includes ceph-osd as dependency.